app/views/placements/wizards/add_placement_wizard/_preview_placement_step.html.erb

```diff
@@ -0,0 +1,3 @@
+<%= content_for :page_title, sanitize(t(".page_title", contextual_text:)) %>
+
+<%= render "placements/placements/placement_details", school: @wizard.school.decorate, placement: preview_placement_step.placement.decorate, preview: true %>
```

Comment:
Move "important" panel directly to this file, and then remove the `preview: true` param from the partial.